### PR TITLE
[DOCS] technical terms create expectations: core skills

### PR DESCRIPTION
**Pull request by Rachel-Reverie**
Please annotate your PR title to describe what the PR does, then give a brief bulleted description of your PR below. PR titles should begin with [BUGFIX], [FEATURE],  [DOCS], or [MAINTENANCE]. If a new feature introduces breaking changes for the Great Expectations API or configuration files, please also add [BREAKING]. You can read about the tags in our [contributor checklist](https://docs.greatexpectations.io/en/latest/contributing/contribution_checklist.html).

Changes proposed in this pull request:
- Adds technical term tags to the "Create Expectations: Overview" document
- Adds technical term tags to all documents under "Create Expectations: Core skills" in the ToC
- Edits all of the above documents to bring them in line with our documentation and how-to guide standards.

After submitting your PR, CI checks will run and @ge-cla-bot will check for your CLA signature.

For a PR with nontrivial changes, we review with both design-centric and code-centric lenses.

In a design review, we aim to ensure that the PR is consistent with our relationship to the open source community, with our software architecture and abstractions, and with our users' needs and expectations. That review often starts well before a PR, for example in github issues or slack, so please link to relevant conversations in notes below to help reviewers understand and approve your PR more quickly (e.g. `closes #123`).

Previous Design Review notes:
-
-
-

### Definition of Done
Please delete options that are not relevant.

- [X] I have made corresponding changes to the documentation

Thank you for submitting!

### CODE REVIEW

File: docs/guides/expectations/create_expectations_overview.md [hunk 1]
```diff
@@ -23,33 +24,33 @@ There are a few workflows you can potentially follow when creating Expectations.
...
+As with creating Expectations interactively, you will start with your Data Context.  However, in this case you will be working in a Python environment, so you will need to loading or create your Data Context as an instantiated object.  Next, you will create a Batch Request to specify the data you would like to <TechnicalTag tag="profiling" text="Profile" /> in order to create your Expectation Suite.  Once you have a <TechnicalTag tag="batch_request" text="Batch Request" /> configured you will use it and the name for your Expectation Suite to instantiate a Validator.  The Validator will be used to instantiate a Profiler, and you will then use the `build_suite()` method of the Profiler to create your Expectation Suite.
```
Comment (rdodev) [inline docs/guides/expectations/create_expectations_overview.md]: > so you will need to loading or create

is the gerund intended?

Comment (Rachel-Reverie) [reply docs/guides/expectations/create_expectations_overview.md]: It is not.  I'll correct that.

File: docs/guides/expectations/how_to_create_and_edit_expectations_based_on_domain_knowledge_without_inspecting_data_directly.md [hunk 2]
```diff
@@ -3,124 +3,121 @@ title: How to create and edit Expectations based on domain knowledge, without in
...
+3. You know exactly how you want your <TechnicalTag tag="expectation" text="Expectations" /> to be configured.
 4. You want to create Expectations parametrically (you can also do this in interactive mode).
-
 5. You don't want to spend the time to validate against a sample.
 
+If you have a use case we have not considered, please [contact us on Slack](https://greatexpectations.io/slack).
 
 <Prerequisites>
 
 - Configured a [Data Context](../../tutorials/getting_started/initialize_a_data_context.md).
```
Comment (rdodev) [inline docs/guides/expectations/how_to_create_and_edit_expectations_based_on_domain_knowledge_without_inspecting_data_directly.md:22]: I wonder if adding qualifiers would help here: "A configured and defined Data Context"

File: docs/guides/expectations/create_expectations_overview.md (hunk 1 above)
Comment (Rachel-Reverie) [reply docs/guides/expectations/create_expectations_overview.md]: Fixed.

**Review by rdodev — APPROVED** [review]

File: docs/guides/expectations/how_to_create_and_edit_expectations_based_on_domain_knowledge_without_inspecting_data_directly.md (hunk 2 above)
Comment (Rachel-Reverie) [reply docs/guides/expectations/how_to_create_and_edit_expectations_based_on_domain_knowledge_without_inspecting_data_directly.md:22]: I'm uncertain what the benefit of those qualifiers would be.  We're already specifying that they need to have configured a Data Context, and the next line specifies a specific way in which it needs to be configured.  How do you see "defined" differing from/adding to "configured" in this scenario?

**Pull request merged**